compiler/one-cmds/one-prepare-venv.u1804.aarch64

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
As I remember, this file will be copied as `one-prepare-venv.u1804` with
`get_filename_component(ONE_COMMNAD_FILE_NAME ${ONE_COMMAND} NAME_WE)` line.
Can you check again with copied file name?

FYI, I proposed to use `one-prepare-venv.u1804-aarch64`

Reply:
My local build shows this is OK.

Reply:
> FYI, I proposed to use `one-prepare-venv.u1804-aarch64`

Oops, my mistake.